src/vcfio/vcf_writer.py

```diff
@@ -41,10 +41,24 @@ def write_headers(self, headers: Iterable[AnyStr] = ()):
         if not headers:
             headers = [REQUIRED_HEADERS[0]]
             self._write_last_header = True
-        for header in headers:
+        sorted_headers = self.sort_headers(headers)
```

Comment:
could we move the custom_sort into a private member function of  the class and call it like this? 
```suggestion
        sorted_headers = sorted(headers, , key=self.__sort_headers))
```